spec/problem_one_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../problem_one'
+
+describe ProblemOne do
+  it 'computes for a set of even numbers' do
```

Comment:
This is for all the specs. 
It should be clear what method you are testing, what is the context that you are testing for, and what is the actual expectation.
Check RSpec `describe` method, `context` method.

Reply:
This is available for all specs